internal/alertmanager/dispatcher/dispatcher.go

```diff
@@ -52,7 +52,7 @@ func alertFilter(promAlerts []model.Alert) []model.Alert {
 
 	filteredAlerts := []model.Alert{}
 	for _, alert := range promAlerts {
-		if _, labelOK := alert.Labels["kafka_cr"]; labelOK {
+		if label, labelOK := alert.Labels["alertGroup"]; labelOK && label == "kafka" {
```

Comment:
This doesn't look to be backward compatible with  previous deployments. I'd do the filtering based on command.